pkgs/development/python-modules/paramiko/default.nix

```diff
@@ -18,12 +18,12 @@
 
 buildPythonPackage rec {
   pname = "paramiko";
-  version = "3.4.1";
+  version = "3.5.0";
   format = "setuptools";
 
   src = fetchPypi {
     inherit pname version;
-    hash = "sha256-ixUwKHCvf2ZS8uA4l1wdKXPwYEbLXX1lNVZos+y+zgw=";
+    hash = "sha256-rRHlQNpPVc7dpSkx8aP4Eqgjinr39ipg3lOM2AuygSQ=";
   };
 
```

Comment:
Are you interested in becoming a maintainer for this package? No one is currently maintaining it. If you're not interested, sorry to bother you, could you add me (momeemt) to the maintainer list (line 79) as follows `maintainers = with lib.licenses.maintainers [ momeemt ];`?

Reply:
we are fine with maintaining it, doing this kinda already. i've also done the last bump